python/ray/dashboard/modules/job/common.py

```diff
@@ -370,12 +370,7 @@ async def get_job_info(job_id: str):
             job_info = await self.get_info(job_id, timeout)
             return job_id, job_info
 
-        return {
-            job_id: job_info
-            for job_id, job_info in await asyncio.gather(
-                *[get_job_info(job_id) for job_id in job_ids]
-            )
-        }
+        return dict(await asyncio.gather(*[get_job_info(job_id) for job_id in job_ids]))
```

Comment:
The CI is currently failing. I think some of your fixes might be incorrect and need to be reviewed more carefully. For example, this line is a loop over awaitables, which is different from other places where it loops over dictionary items.